.dockerignore

```diff
@@ -62,4 +62,5 @@
 !rust/datafusion/benches
 !rust/integration-testing/Cargo.toml
 !go/go.mod
-!go/go.sum
\ No newline at end of file
+!go/go.sum
+!swift/Arrow
```

Comment:
I think that this is needless because this is not used in Dockerfile. We can access `swift/Arrow` in `docker run` because we mount the source directory to `/arrow` in `docker run`.

Reply:
Gocha, will remove.

Reply:
I tried removing the above but it fails with the below error when I run 'archery docker run ubuntu-swift bash'

` > [2/3] ADD swift/ /arrow/swift:
------
failed to compute cache key: "/swift" not found: not found
ERROR: Service ubuntu-swift failed to build : Build failed
Error: docker-compose --file /Users/alvabandy/Documents/GitHub/arrow/docker-compose.yml build --build-arg BUILDKIT_INLINE_CACHE=1 ubuntu-swift exited with a non-zero exit code 1, see the process log above.`

Reply:
We can remove `ADD swift/ /arrow/swift` because we mount `swift/` in `docker run`.

Reply:
Ok, will remove.